File: 389-find-the-difference/find-the-difference.java
class Solution {
    public char findTheDifference(String s, String t) {
        int len1=s.length();
        int sum=0;
        for(int i=0;i<len1;i++){
            sum -=s.charAt(i);
            sum +=t.charAt(i);
        }
        sum+=t.charAt(len1);
        return (char)(sum);
    }
}

File: 389-find-the-difference/find-the-difference.c
char findTheDifference(char* s, char* t) {
    int s_size= strlen(s);
    int sum=0;
    for(int i=0;i<s_size;i++){
        sum-= *s;
        s++;
        sum+= *t;
        t++;
    }
    sum+=*t;
    return sum;
}